src/nvim/terminal.c

```diff
@@ -172,6 +173,39 @@ void terminal_teardown(void)
   pmap_init(ptr_t, &invalidated_terminals);
 }
 
+static int on_osc_fallback(const char *command, size_t cmdlen, void *user)
```

Comment:
I would rename this function parse_osc in order to match the [name used by Vim](https://github.com/vim/vim/blob/af59e34f1bf214ec6f2d0304c1c5b11ecda25a83/src/terminal.c#L4466), which could potentially make porting future patches easier

Reply:
Done (f30cc5376df62449956f9700afa903c689c7986b).

Reply:
Done (f30cc5376df62449956f9700afa903c689c7986b).

Reply:
A downside to the name `parse_osc` is that the function doesn't parse its input.

Even prior to the change of function signature to satisfy the `libvterm` update, the function was seemingly not properly described by the name `parse_osc`. At the time, the function was checking if the OSC command string was of a particular command integer, and responding accordingly. Now `libvterm` passes the OSC command as an integer directly.